### PR TITLE
#845 Clarifying Inactive Vs Offline Devices

### PR DESCRIPTION
**Pull request by Lizzieepton**
Added paragraph to clarify the billing differences between inactive and offline devices
Issue #845: https://github.com/balena-io/docs/issues/845

Change-type: minor
Signed-off-by: Lizzie Epton <lizzie@balena.io>

### CODE REVIEW

File: pages/learn/manage/billing.md [hunk 1]
```diff
@@ -65,3 +65,7 @@ Marking a device as `Inactive` is a [device action][device-action]. Device actio
 [device-action]:/learn/manage/actions/#device-specific-actions
 [inactive]:#inactive-devices
 [organizations]:/learn/manage/organizations/
+
+## Inactive vs Offline Devices
+
+If you have a device offline-but-active you will still be billed for that device. Those devices are still deployed in the field and ready to be used at any time, if you make the decision to have your devices offline intentionally, you will still have full visibility to remotely monitor the device and are able to update it. You will not be able to monitor inactive devices and won't be charged for them after the de-activation fee until you reactive them on balenaCloud.
```
Comment (Bucknalla) [inline pages/learn/manage/billing.md:71]: Hey this great but I think we could explain offline and inactive as a comparative to make it a little more clear

**Review by Bucknalla — APPROVED** [review]
LGTM

**Comment by Bucknalla** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
Resolves #845 GitHub won't let me link it back.

**Pull request merged**